docker/oss-fuzz/host/start_host.py

```diff
@@ -14,9 +14,6 @@
 """Start host."""
 from __future__ import print_function
```

Comment:
Should all these from __future__ go away too.

Reply:
Will do this in another PR :)